guardrails/guardrail_globals.py

```diff
@@ -3,6 +3,9 @@
 import sys
 from configparser import ConfigParser
 import os
+import glob
+# import glob2 # for python 2.7
```

Comment:
Why is this import commented please?

Reply:
rephrased the comment, glob2 is meant for python 2.7